src/components/search/results/results.vue

```diff
@@ -2,7 +2,12 @@
   <ResultsList v-if="$x.totalResults" v-infinite-scroll:main-scroll>
     <BannersList>
       <PromotedsList>
-        <NextQueriesList :offset="24" :frequency="48" :maxNextQueriesPerGroup="1">
+        <NextQueriesList
+          :offset="24"
+          :frequency="48"
+          :maxNextQueriesPerGroup="1"
+          :showOnlyAfterOffset="$x.partialResults.length === 0"
```

Comment:
I think that this condition should be inverted, because what we want to achieve is to show the NQ Previews when the results number is lesser than the offset:

```suggestion
          :showOnlyAfterOffset="$x.partialResults.length > 0"
```

Reply:
I agree with @ajperezbau, I would invert the condition too